### DISCUSSION TITLE
HydroMT is unable to infer flow direction file

### DISCUSSION THREAD

**Discussion by AldenShadd1000** [Deltares/Wflow.jl#485] [Q&A] [answered]
I generated a flow direction file from a DEM of resolution of 90 metres using the code provided at Hydromt website: (https://deltares.github.io/hydromt_wflow/latest/_examples/prepare_ldd.html) But, I encountered the following error:
![Error_Screenshot](https://github.com/user-attachments/assets/4cf65a22-d80d-43b7-bd88-89942a569061)

I am attaching the log file below.

[hydromt.log](https://github.com/user-attachments/files/17409284/hydromt.log)

Kindly guide on how to proceed on this.

**Comment by verseve** [comment, marked as answer]
Thanks for reporting this issue. Could you please report this issue here:
https://github.com/Deltares/hydromt_wflow/issues

This repository is about the Wflow kernel, for hydromt_wflow (model building issues) please report through the link above.
